python/lsst/meas/modelfit/cmodel/cmodelContinued.py

```diff
@@ -21,18 +20,24 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 
-import lsst.pex.config
+from .cmodel import CModelStageControl, CModelControl, CModelAlgorithm
+
+from lsst.pex.config import makeConfigClass
 import lsst.meas.base
-from . import modelfitLib
 
+__all__ = ("CModelStageConfig", "CModelConfig")
```

Comment:
The code in this particular file is a happy exception to the "mostly untested" statement.  The plugin classes are added to a registry in `lsst.meas.base` (via the `register` decorator), and are used exclusively from that registry, so there's no need to import them directly from this module.  I see that I added a comment to that effect in `psfContinued.py` but not here; I'll add one here, too.